terrawrap/utils/cli.py

```diff
@@ -101,7 +112,15 @@ def execute_command(
         time_passed = jitter.backoff()
 
     if audit_api_url and kwargs['cwd']:
-        _post_to_audit_api_url(audit_api_url, kwargs['cwd'], exit_code, stdout)
+        # Call _post_audit_info again, this time to update the 'in progress' entry with new status and output
+        _post_audit_info(
+            audit_api_url=audit_api_url,
+            path=kwargs['cwd'],
+            exit_code=exit_code,
+            stdout=stdout,
+            timestamp=timestamp,
```

Comment:
what format is the timestamp supposed to be? is this suppose to be human readable?

Reply:
also, you need to create a new timestamp value for this call

Reply:
what is timestamp suppose to be here? is it supposed to be the start of this call? or is is supposed to be the start of `IN PROGRESS` (prev call above)?

Reply:
Timestamp is supposed to be the same for each call (start of `IN PROGRESS`). 

In order for me to update an element in DynamoDB via the API, I need to know both the hash and range key of said element. To accomplish this, I now have timestamp set by whoever is calling the API in the request body (terrawrap, in this case). Hence timestamp being set at the beginning of `execute_command` and that same value being used by both `_post_audit_info` calls.

Reply:
i misunderstood the purpose of the timestamp for this call. i don't think you should be sending it at all for this call. take a look at comments here: https://github.com/amplify-education/terraform-audit-api/pull/14/files#r888455930

Reply:
Sorry for the word vomit, couldn't really come up with a better way to implement the "update" functionality